metabase/metabase
# Bookmarks: render official icons for collections

### How to Test

1. Run application in enterprise mode: `yarn dev-ee` 
2. Add token in Admin › Billing
3. Create
4. As you save it, make it an "Official" collection
5. Bookmark this collection

In the sidebar, you should see this collection in the bookmarks list, with the official badge.

<img width="500" alt="image" src="https://user-images.githubusercontent.com/380816/160031041-dea4dad8-0c9c-4abe-bcac-b175bd647445.png">
